src/test-routes/media-types.ts

```diff
@@ -36,4 +36,27 @@ app.category("vanilla", () => {
       };
     }
   });
+  app.post("/mediatypes/binaryBodyThreeContentTypes", "MediaTypesThreeContentTypes", (req) => {
```

Comment:
i think this looks good, have you verified the test works locally? Also can you bump the package.json version number, we bump the version number for each test added to trigger a new release and allow people to pin to prior version numbers if necessary

Reply:
I have now 😅 I had to test the text/plain route manually because autorest.go doesn't generate code for it; turns out express needed parser config for it as well.